test/scf_compare.jl

```diff
@@ -86,7 +90,11 @@ end
     # Run Newton algorithm
     if mpi_nprocs() == 1  # Distributed implementation not yet available
         @testset "Newton" begin
-            ψ0 = self_consistent_field(basis, maxiter=1).ψ
+            scfres_start = self_consistent_field(basis, maxiter=1)
+            # remove virtual orbitals
```

Comment:
Hm, should we have a helper function to do this? I'm not sure where to put it... Let's wait until we have some restructuring to do